operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 * [ENHANCEMENT] Store-gateway: set `GOMEMLIMIT` to the memory request value. This should reduce the likelihood the store-gateway may go out of memory, at the cost of an higher CPU utilization due to more frequent garbage collections when the memory utilization gets closer or above the configured requested memory. #4971
 * [ENHANCEMENT] Store-gateway: dynamically set `GOMAXPROCS` based on the CPU request. This should reduce the likelihood a high load on the store-gateway will slow down the entire Kubernetes node. #5104
 * [ENHANCEMENT] Add global.podLabels which can add POD labels to PODs directly controlled by this chart (mimir services, nginx). #5055
+* [ENHANCEMENT] Enable the `track_sizes` feature for Memcached pods. #5209
```

Comment:
Can you mention what this means for operators? I assume 1) an increase in metrics generated by the exporter and 2) insight into distribution of item sizes? I'm not sure what I wrote even makes sense, but would be good to give some context to the operators reading the changelog.

I suppose the same comment applies to the jsonnet changelog

Reply:
No increase in metrics. It's just a histogram for keeping track of items that you can inspect with `stats sizes` when connected to Memcached. It really is just enabling a feature from the perspective of operators. The idea is that we (Grafana) could potentially adjust how we're storing things in Mimir in the future to make the most of the space being used by Memcached. E.g. we could adjust the max subrange size for the chunks cache to fit almost perfectly into a slab instead of the case today were there's about 1K of wasted space per item.